### PR TITLE
Add linking API endpoints

### PR DESCRIPTION
**Pull request by chloepouprom**
@jugarrit @lemieux @dannytranlx @mspensieri 

This adds the endpoints for
`POST /appUsers/:userId/channels`
`DELETE /appUsers/:userId/channels/:channel`

### CODE REVIEW

File: src/api/appUsersLink.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        const url = this.getFullURL('appUsers', userId, 'channels');
+        return this.request('POST', url, data);
+    }
+
+    /**
+     * Unlinks the specified channel
+     * @param {string} userId - a user id
+     * @return {APIResponse}
+     */
+    deleteTwilioChannel(userId) {
```
Comment (jugarrit) [inline src/api/appUsersLink.js]: Should this just be `deleteChannel` perhaps?

```js
deleteChannel(userId, channel) {
    const url = this.getFullURL('appUsers', userId, 'channels', channel);
    return this.request('DELETE', url);
}
```

**Comment by jugarrit** [comment]
Rest looks good 👍

**Comment by mspensieri** [comment]
👍

**Comment by lemieux** [comment]
:+1:

File: src/api/appUsersLink.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        return this.request('POST', url, data);
+    }
+
+    /**
+     * Unlinks the specified channel
+     * @param {string} userId - a user id
+     * @param {string} channel - the channel to unlink
+     * @return {APIResponse}
+     */
+    deleteChannel(userId, channel) {
```
Comment (lemieux) [inline src/api/appUsersLink.js]: why isn't this called `unlinkChannel`? Even your doc just above says it will unlink the channel.

Comment (chloepouprom) [inline src/api/appUsersLink.js]: good point. I'll rename

**Pull request merged**